### ISSUE TITLE
Understanding document for “Pointer Target Spacing”, a non-existent WCAG 2.2 SC, is still available

### ISSUE THREAD

**Issue by yatil** [w3c/wcag#1833]
This page still exists: https://www.w3.org/WAI/WCAG22/Understanding/pointer-target-spacing.html
This is the correct page: https://www.w3.org/WAI/WCAG22/Understanding/target-size-minimum.html

Expectation: Understanding documents for deleted/renamed or renamed SCs are removed and no longer findable through search engines. If they are renamed they redirect reliably to the new name. No old information is shown.

People are actively finding these outdated pages. From an accessibility slack (emphasis mine):

> I got **supremely confused** reading the old WCAG version of the “Target Size” page (was titled “Pointer Target Spacing”) that was full of mislabeled diagrams and math that didn’t add up. 💫 Only after thoroughly studying that page, did I later find the new page, linked above. But now I’m already turned around and questioning everything haha

The confusion that is created through errors in the understanding documents creates major disruption. There are many open issues available that describe the issues. They need to be fixed immediately. It harms the reputation of the WCAG documents as a whole. I stopped referring clients to them because they are so incredibly unreliable. I try to pretend that they do not exist at this point. That's basically the opposite from what I did from before WCAG 2.1.

**Comment by jake-abma** [comment]
I understand where you come from and agree on possible deletion / renaming, but also the page has a clear Status message:

<img width="721" alt="Screen Shot 2021-05-31 at 9 13 43 AM" src="https://user-images.githubusercontent.com/6384002/120155126-d1bbf500-c1f0-11eb-97d0-bcfb257fbe85.png">

> Status
> This understanding document is part of the draft WCAG 2.2 content. It may change or be removed before the final WCAG 2.2 is published.

This should prevent comments and inform people, so they are not to be "confused" and is almost at the top of the page so you do not need to "thoroughly studying that page"

Do you have other ideas for these kind of messages so they work better?

**Comment by yatil** [comment]
@jake-abma The issue here is not that someone searched for WCAG 2.0 or 2.1 information, they explicitly searched for WCAG 2.2 information. This is a page that should not exist because it will not be in WCAG 2.2 and all of its content has been ported to the _Target Size Minimum_ page and is improved there. I don’t know how that person stumbled over this particular page.

I also take issue with “People should not review WCAG 2.2 pages they find because it is a draft and could be removed” despite WCAG 2.2 being in [active wide review](https://www.w3.org/blog/news/archives/9052). Shouldn’t AG WG encourage people interested in WCAG to review these documents and give feedback? And if AG WG want to encourage that, they should not waste people’s time with outdated, deleted information.

**Comment by yatil** [comment]
Especially considering that the correct page has the exactly same status message. How are readers supposed to know?

**Comment by alastc** [comment]
@michael-n-cooper Shouldn't this page be removed with a fresh update of the 2.2 docs?
